### PR TITLE
Forget password

### PR DESCRIPTION
**Pull request by aimanzi**
hi team 4 👍
sorry for the dual work i made mistake with push to uesed branchs which caused a conflicts between currently branch
for you check
its the same files that you confirm and marged

### CODE REVIEW

File: Pages Files/forgetpassword.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+<body>
+    <div class="header">
+        <h1>Forgot My Password</h1>
+    </div>
+    <div class="tofes">
+        <form>
+            <label for="fname">What is your phone number ?</label><br>
+            <input class="inputs" type="tel" id="phone" name=""><br>
+            <label for="lname">What is your email ?</label><br>
+            <input class="inputs" type="email" id="email"><br><br>
```
Comment (nevensh) [inline Pages Files/forgetpassword.html:24]: hey team 3. 
why you use br tag ? in return you can use div 
every label or every input be inside " div " .
the best way to do a line break .

**Review by nevensh — CHANGES_REQUESTED** [review]
please fix it

**Comment by aimanzi** [comment]
conflict is solved

**Comment by aimanzi** [comment]
un active branch "ForgetPassword"

**Comment by aimanzi** [comment]
activatinf branch

**Pull request merged**